spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResourceBucket.java

```diff
@@ -111,10 +122,15 @@ public long lastModified() {
 
 	@Override
 	public Resource createRelative(String relativePath) throws IOException {
```

Comment:
Please change the return type to `GoogleStorageResourceObject`.

Reply:
Regarding `createRelative()`, the behaviour is slightly different between `GoogleStorageResourceBucket` and `GoogleStorageResourceObject`.

Whereas the latter creates a sibling blob, the former doesn't create a sibling bucket but rather a blob within that bucket.

Not sure what to do, if anything, about this...